pkg/kudoctl/util/kudo/install.go

```diff
@@ -27,7 +28,12 @@ func InstallPackage(kc *Client, resources *packages.Resources, skipInstance bool
 		var manifest string = ""
 		if resources.Operator.Spec.NamespaceManifest != "" {
 			clog.V(3).Printf("creating namespace with manifest named: %q", resources.Operator.Spec.NamespaceManifest)
-			manifest = resources.OperatorVersion.Spec.Templates[resources.Operator.Spec.NamespaceManifest]
+			template := resources.OperatorVersion.Spec.Templates[resources.Operator.Spec.NamespaceManifest]
+			var err error
+			manifest, err = render("namespace", template, resources, instanceName, namespace, parameters)
+			if err != nil {
+				return err
```

Comment:
```suggestion
				return fmt.Errorf("failed to render namespace manifest %s: %v", resources.Operator.Spec.NamespaceManifest, err)
```

Might be good to add some context to the error here?